README.md

```diff
@@ -138,6 +138,7 @@ Runs COBOL code from Node.JS side.
 - **Function|String|Path** `input`: A function containing a comment with inline COBOL code, the cobol code itself or a path to a COBOL file.
 - **Object** `options`: An object containing the following fields:
  - `cwd` (String): Where the COBOL code will run (by default in the current working directory)
+ - `free` (Boolean): Use free option while compiling with GnuCobol
```

Comment:
Oh, and [add this line in the code too](https://github.com/IonicaBizau/node-cobol/blob/fbbe2d25cbd8ec7c7435fab9053ce5943fe577e0/lib/index.js#L23). Actually the README.md file is generated via Blah. I wrote on my blog [how I use Blah to generate docs](http://ionicabizau.net/blog/25-increasing-productivity-using-blah-2-0-0). :star2: